### ISSUE TITLE
Scale animation to fit parent container

### ISSUE THREAD

**Issue by Psli** [tfmart/LottieUI#3]
```
HStack {
  LottieView("loading").loopMode(.loop)
}.frame(width: 200, height: 200, alignment: .center)
```
See here for solution?
 https://github.com/airbnb/lottie-ios/issues/1050

**Comment by tfmart** [comment]
Hi @Psli, thanks for reporting this issue. Could you link me the animation you're trying to use so I can try to reproduce it? I haven't run into this on my project at least. 

In the meantime, if you end up making some changes to the library that fixes this issue, feel free to let me know!

**Comment by Psli** [comment]
This is a test project https://github.com/Psli/TestLottieUI.git
LottieView(...).frame(width: 200, height: 200, alignment: .center) not working

**Comment by tfmart** [comment]
Thanks for the demo, I'll give a looks and see if I can sort it out

**Comment by tfmart** [comment]
Hey @Psli, I've submitted a PR that may fix your issue. Could you try using the `mac-scaling-fix` branch on your project and check if the issue persists?

**Comment by Psli** [comment]
Hey @tfmart , the PR fix my issue. Thanks!

**Comment by tfmart** [comment]
Great! I've merged the branch and [released a new version 1.3.1](https://github.com/tfmart/LottieUI/releases/tag/1.3.1) which contains this patch, so I'll be closing this issue. If you run into any other problem, feel free to reach out!
